packages/shared/dao/index.ts

```diff
@@ -185,7 +185,8 @@ async function resolveOfflineRealmAboutFromConnectionString(
         healthy: true,
         lambdas: {
           healthy: true,
-          publicUrl: `${baseUrl}lambdas`
+          publicUrl: `${baseUrl}lambdas`,
+          publicUrl2: `${baseUrl}lamb2`
```

Comment:
This is not supposed to happen, instead lamb2 should be accesible via nginx re-route to lambdas @manumena please confirm

Reply:
I will be happy to remove the whole PR if it works this way.

Reply:
How can it work this way considering that the names of endpoints are the same (e.g. "profiles") for old and new API, and we will go through the feature flag at first?

Reply:
@menduz @mikhail-dcl  lamb2 doesn't serve 3rd party wearables and can't be filtered by collectionId yet. The plan was
1. Focused to support the extended passport scope (pagination and other features)
2. Add 3rd party, collectionId and whatever is necessary to reach current functionality
3. Re-route lambdas to lamb2, keeping both endpoints available
4. Deprecate lambdas

We haven't started step 2 yet

Reply:
@menduz 
the paths today would be like
- `/lambdas/wearables-by-owner` hit lambdas
- `/lamb2/nfts/wearables` hit lamb2

If your concern is to maintain the same service name for clarity reasons, we can do the re-routing right now and leave the endpoints like 
- `/lambdas/wearables-by-owner` hit lambdas
- `/lambdas/nfts/wearables` hit lamb2

and in another iteration

- `/lambdas/wearables-by-owner` hit lamb2
- `/lambdas/nfts/wearables` hit lamb2

so the name `lamb2` is not exposed

Reply:
I'd like to not add extra configuration settings to all projects and move forward with the route of lamdas pointing to lamb2 to be consistent with the lamb2 migration plan

Reply:
@manumena @pentreathm will you have time to work on routing shortly? At the moment it will block us to proceed further if we want to stick with a single path.

Reply:
yes, it's a small change. 
@manumena all yours :)

Reply:
I'll start immediately